tests_e2e/test_suites/agent_publish.yml

```diff
@@ -0,0 +1,11 @@
+#
+# This test is used to verify that the agent will be updated after publishing a new version to the agent update channel.
+#
+name: "AgentPublish"
+tests:
+  - "agent_publish/agent_publish.py"
+images:
+  - "random(endorsed, 10)"
+  - "random(endorsed-arm64, 2)"
```

Comment:
No need to test it in all distros as this is not distro dependent validation

Reply:
sounds good, should we do the same for the agent update test?

Reply:
yeah, same thing applies to agent update test